src/ralph/data_importer/management/commands/importer.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+
+from django.apps import apps
+from django.core.management.base import BaseCommand
+from import_export import resources
+from ralph.data_importer import resources as ralph_resources
+from ralph.lib import unicode_csv
+from six.moves import input
+import tablib
+
+
+APP_MODELS = {model._meta.model_name: model for model in apps.get_models()}
+
+
+def get_resource(model_name):
+    """Return resource for import model."""
+    resource_name = model_name + 'Resource'
+    resource = getattr(ralph_resources, resource_name, None)
+    if not resource:
+        model_class = APP_MODELS[model_name.lower()]
+        resource = resources.modelresource_factory(model=model_class)
+    return resource()
+
+
+class Command(BaseCommand):
+
+    help = 'Imports data for specified model from specified file'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--model_name', default=False, help='TODO::',
```

Comment:
is this param required? if yes, it should be not only `--` param, but `-m` too. What about default - why `False`, not `None`? And fix TODO :)